controllers/rhmi/rhmi_controller.go

```diff
@@ -1095,7 +1095,7 @@ func (r *RHMIReconciler) preflightChecks(installation *rhmiv1alpha1.RHMI, instal
 	}
 	if isSTS {
 		log.Info("validation of STS role ARN parameter ")
-		validArn, err := sts.ValidateAddOnStsRoleArnParameterPattern(r.Client, installation.Namespace)
+		validArn, err := checkStsCredentialsPresent(r.Client, installation.Spec.NamespacePrefix+"-cloud-resources-operator")
```

Comment:
```suggestion
		validArn, err := checkStsCredentialsPresent(r.Client, installation.Spec.NamespacePrefix+"cloud-resources-operator")
```
I think the prefix usually contains the `-`

Reply:
Are we losing the validate function here?

Reply:
yea have it change locally

I don't think so this check was originally for when we were passing in an arn rather that letting managed tenants create the arn.

Reply:
going to remove this functionally as it relied on CRO ns being created by managed tenants/hive